### PR TITLE
feat: add prevent tx test

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/8047835811), [Test report](https://leather-wallet.github.io/playwright-reports/feat/add-prevent-tx-test)

This pr adds prevent transaction test in case when tx contains inscribed utxo
Isssue: https://github.com/leather-wallet/extension/issues/4920

### CODE REVIEW

File: src/app/query/bitcoin/transaction/use-check-utxos.ts [hunk 1]
```diff
@@ -100,8 +100,9 @@ export function useCheckInscribedUtxos(blockTxAction?: () => void) {
 
       try {
         // no need to check for inscriptions on testnet
-        if (isTestnet) {
-          return false;
+        // TO_DO how to handle this in playwright mode?
+        if (!isTestnet) {
+          // return false;
```
Comment (alter-eggo) [inline src/app/query/bitcoin/transaction/use-check-utxos.ts]: wonder how to better handle this... add some special `IS_PLAYWRIGHT_ENV` variable?

Comment (kyranjamie) [reply src/app/query/bitcoin/transaction/use-check-utxos.ts]: WALLET_ENVIRONMENT === 'testing'?

Comment (alter-eggo) [reply src/app/query/bitcoin/transaction/use-check-utxos.ts]: I'm thinking of creating new env, like IS_PLAYWRIGHT_ENV, as we limit the app in test env, e.g.
```
  if (!IS_TEST_ENV)
    persistQueryClient({
      queryClient,
      persister: localStoragePersistor,
      buster: VERSION,
    });
```

Comment (kyranjamie) [reply src/app/query/bitcoin/transaction/use-check-utxos.ts]: Playwright _is_ test env? Why do we need a new one?

Comment (alter-eggo) [reply src/app/query/bitcoin/transaction/use-check-utxos.ts]: currently in playwright mode `WALLET_ENVIRONMENT` is `unknown`

Comment (alter-eggo) [reply src/app/query/bitcoin/transaction/use-check-utxos.ts]: ah, it's unknown locally, on ci it's testing

Comment (alter-eggo) [reply src/app/query/bitcoin/transaction/use-check-utxos.ts]: added check on testing env

File: tests/mocks/mock-ordinalscom-api.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+export const mockOrdinalscomApiHtmlResponse = `
```
Comment (fbwoolf) [inline tests/mocks/mock-ordinalscom-api.ts]: Just noticed in other PR you typ used `...OrdinalsCom...` so noting here you might want to capitalize `Com` to make code searches consistent?

**Review by fbwoolf — APPROVED** [review]

**Pull request merged**